File: Src2/Icm.Core/Localization/IPhraseExtensions.vb
Imports System.Runtime.CompilerServices

Namespace Icm.Localization

    Public Module IPhraseExtensions

        <Extension>
        Public Function Translate(phrase As IPhrase, ByVal locRepo As ILocalizationRepository) As String
            Return phrase.Translate(CultureInfo.CurrentCulture.LCID, locRepo)
        End Function

    End Module

End Namespace

File: Src/Icm.Core/Localization/IPhraseExtensions.cs
using System.Runtime.CompilerServices;

namespace Icm.Localization
{

	public static class IPhraseExtensions
	{

		[Extension()]
		public static string Translate(IPhrase phrase, ILocalizationRepository locRepo)
		{
			return phrase.Translate(CultureInfo.CurrentCulture.LCID, locRepo);
		}

	}

}

//=======================================================
//Service provided by Telerik (www.telerik.com)
//Conversion powered by NRefactory.
//Twitter: @telerik
//Facebook: facebook.com/telerik
//=======================================================
